diff --git a/Microsoft.PowerShell_profile.ps1 b/Microsoft.PowerShell_profile.ps1
index 2f7f86b..f65ab61 100644
--- a/Microsoft.PowerShell_profile.ps1
+++ b/Microsoft.PowerShell_profile.ps1
@@ -17,10 +17,10 @@ Set-PsFzfOption -EnableAliasFuzzyEdit -PSReadLineChordProvider 'Ctrl+f' -PSReadL
 
 # Set Some Option for PSReadLine to show the history of our typed commands
 Import-Module -Name CompletionPredictor
-Set-PSReadLineOption -PredictionSource HistoryAndPlugin
-Set-PSReadLineOption -PredictionViewStyle ListView
-Set-PSReadLineOption -EditMode Windows
-# Set-PSReadLineOption -PredictionSource None
+# Set-PSReadLineOption -PredictionSource HistoryAndPlugin
+# Set-PSReadLineOption -PredictionViewStyle ListView
+# Set-PSReadLineOption -EditMode Windows
+Set-PSReadLineOption -PredictionSource None
 
 # Utility Command that tells you where the absolute path of commandlets are
 function which ($command) {
